common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp

```diff
@@ -94,29 +94,15 @@ SteeringAngleDisplay::~SteeringAngleDisplay()
 
 void SteeringAngleDisplay::onInitialize()
 {
-  RTDClass::onInitialize();
+  MFDClass::onInitialize();
```

Comment:
I think I said that we can switch back to MFDClass in the previous meeting, but after having second thought, I think we can keep it as RTDClass. The change on moving visualization process to `update` function from `processMessage` can still be applied for stability and process time.
I apologize for going back and forth, but could you make the super class to RTD?

Reply:
in all the plugins?